test/position-managers/IncreaseLiquidity.t.sol

```diff
@@ -335,4 +335,41 @@ contract IncreaseLiquidityTest is Test, PosmTestSetup, Fuzzers {
             );
         }
     }
+
+    function test_mint_settleWithBalance() public {
+        uint256 liquidityAlice = 3_000e18;
+
+        Plan memory planner = Planner.init();
+        planner.add(Actions.MINT_POSITION, abi.encode(config, liquidityAlice, alice, ZERO_BYTES));
+        planner.add(Actions.SETTLE_WITH_BALANCE, abi.encode(currency0, type(uint256).max));
+        planner.add(Actions.SETTLE_WITH_BALANCE, abi.encode(currency1, type(uint256).max));
+        planner.add(Actions.SWEEP_ERC20_TO, abi.encode(currency0, address(this)));
+        planner.add(Actions.SWEEP_ERC20_TO, abi.encode(currency1, address(this)));
+
+        uint256 balanceBefore0 = currency0.balanceOf(address(this));
+        uint256 balanceBefore1 = currency1.balanceOf(address(this));
+
+        assertEq(currency0.balanceOf(address(lpm)), 0);
+        assertEq(currency0.balanceOf(address(lpm)), 0);
+
+        currency0.transfer(address(lpm), 100e18);
+        currency1.transfer(address(lpm), 100e18);
+
+        assertEq(currency0.balanceOf(address(lpm)), 100e18);
+        assertEq(currency0.balanceOf(address(lpm)), 100e18);
+
+        bytes memory calls = planner.encode();
+
+        vm.prank(alice);
+        bytes[] memory results = lpm.modifyLiquidities(calls, _deadline);
+        uint256 amount0 = abi.decode(results[1], (uint256));
+        uint256 amount1 = abi.decode(results[2], (uint256));
+
+        // The balances were swept back to this address.
+        assertEq(IERC20(Currency.unwrap(currency0)).balanceOf(address(lpm)), 0);
+        assertEq(IERC20(Currency.unwrap(currency1)).balanceOf(address(lpm)), 0);
+
+        assertEq(currency0.balanceOf(address(this)), balanceBefore0 - amount0);
```

Comment:
e.g. here without return values i feel like you can do
```
// you do this already above
assert(balance(lpm), 0)
// new check without return values available
assertEq(changeInPMBalance + changeInThisBalance, 100e18)
```

Reply:
I can get return values now in my other pr so will leave it & fix during merge confs